main/export-options.js

```diff
@@ -50,7 +56,8 @@ const prettifyFormat = format => {
     ['apng', 'APNG'],
     ['gif', 'GIF'],
     ['mp4', 'MP4'],
-    ['webm', 'WebM']
+    ['webm', 'WebM'],
+    ['av1', 'AV1']
   ]);
```

Comment:
```js
const formats = new Map([
  ['apng', 'APNG'],
  ['gif', 'GIF'],
  ['mp4', 'MP4 (H264/HEVC)'],
  ['av1', 'MP4 (AV1)'],
  ['webm', 'WebM']
]);
```

(changed order and titles)

?

Reply:
Wouldn't hevc and h264 be separate options? (when we add hevc)

Reply:
Yes, but it's not added yet.